pymc3/model.py

```diff
@@ -900,14 +900,12 @@ def __new__(cls, *args, **kwargs):
         instance._theano_config = theano_config
         return instance
 
-    def __init__(
-        self, name="", model=None, theano_config=None, coords=None, disable_bounds_check=False
-    ):
+    def __init__(self, name="", model=None, theano_config=None, coords=None, check_bounds=False):
```

Comment:
```suggestion
    def __init__(self, name="", model=None, theano_config=None, coords=None, check_bounds=True):
```
I presume this is what you meant, if for now you only want to rename?